recipe/meta.yaml

```diff
@@ -50,7 +50,7 @@ requirements:
     - scipy >=0.19
     - matplotlib-base >=2.0.0
     - networkx >=2.0
-    - pillow >=4.3.0
+    - pillow >=4.3.0,!=8.3.0
```

Comment:
```suggestion
    - pillow >=4.3.0,<8.3.0
```

Reply:
As the next version is about to be released in a few days https://github.com/conda-forge/pillow-feedstock/pull/96

Reply:
The next version contains the fix linked in my commit.